daemon/src/main/java/org/lsposed/lspd/service/LSPManagerService.java

```diff
@@ -253,11 +253,27 @@ private static int getAutoIncrementNotificationId() {
 
     private static int pushAndGetNotificationId(String modulePackageName, int moduleUserId) {
         var idKey = getNotificationIdKey(modulePackageName, moduleUserId);
-        var idValue = getAutoIncrementNotificationId();
-        notificationIds.putIfAbsent(idKey, idValue);
+        var idValue = notificationIds.get(idKey);
```

Comment:
computeIfAbsent

Reply:
<img width="585" alt="image" src="https://user-images.githubusercontent.com/37344460/164771437-a751bd15-47fd-48cc-bf0e-99903bbbe49d.png">

Reply:
resolve it